src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -109,7 +109,7 @@ limitations under the License.
         <!-- Conformance feature RID - for now optional -->
         <attribute side="server" code="26" define="MIN_RFID_CODE_LENGTH" type="INT8U" writable="false" optional="true">MinRFIDCodeLength</attribute>
         <!-- Conformance feature USR - for now optional -->
-        <attribute side="server" code="27" define="CREDENTIAL_RULES_SUPPORT" type="BITMAP8" min="0x00" max="0x07" default="1" writable="false" optional="true">CredentialRulesSupport</attribute>
+        <attribute side="server" code="27" define="CREDENTIAL_RULES_SUPPORT" type="DlCredentialRule" min="0x00" max="0x07" default="1" writable="false" optional="true">CredentialRulesSupport</attribute>
```

Comment:
This does not match the spec.  In the spec this is a map8, not an enum, and the values of the enum are wrong here.

Reply:
Good catch. Will add DlCredentialRuleMask bitmap and update CredentialRulesSupport's type to DlCredentialRuleMask.